### PR TITLE
Feature/TR-3838/Calculator documentation

### PR DESCRIPTION
**Pull request by jsconan**
### Summary

Add technical documentation.

### Details

- Update the readme file
- Add a page for the API
- Add a page for the history

### How to test

Check the readme and the added pages.

### CODE REVIEW

**Review by rgomeztao — APPROVED** [review]
docs looks good :), approved

File: API.md [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+| `expression`     | `expression`             | Notifies the expression has changed.                            |
+| `position`       | `position`               | Notifies the position inside the expression has changed.        |
+| `variableadd`    | `name`, `value`          | Notifies a variable has been added.                             |
+| `variabledelete` | `name`                   | Notifies a variable has been removed.                           |
+| `variableclear`  |                          | Notifies all variables have been removed.                       |
+| `commandadd`     | `name`                   | Notifies a command has been registered.                         |
+| `commanddelete`  | `name`                   | Notifies a command has been removed.                            |
+| `commandclear`   |                          | Notifies all commands have been removed.                        |
+| `command`        | `name`, ...              | Notifies a command has been invoked.                            |
+| `command-<name>` | ...                      | Notifies the command `<name>` has been invoked.                 |
```
Comment (shaveko) [inline API.md:90]: Please check the usage here https://github.com/oat-sa/live-design-system/blob/9172a8d24d3d9151d2541df121af787b1784bee7/packages/components/calculator/Calculator.svelte#L439

There is space-separation event binding used

Comment (shaveko) [reply API.md:90]: Ok, sorry - it's binding to 2 events in one `on` call

**Review by shaveko — APPROVED** [review]
Only after reading this doc I've realised the actual scale of this task. ..

Nice work and for sure brilliant doc!

**Pull request merged**